### ISSUE TITLE
ESP seems to work but fan does not react to commands

### ISSUE THREAD

**Issue by AllardKatan** [CoMPaTech/esphome_c1101#12]
Hi,
I've triple checked the wiring, testing from the pins on the NodeMCUV3 board to the IC pins on the RF chip, and I can't seem to get this working. The software appears to work, I installed it on the ESP and can see the fan control entities in Home Assistant/ESPhome, and they react to commands. However, I have tried many times to pair the controller to my ventilation system and it doesn't seem to do this at all. Is anything missing from the docs that is very basic but an ESPHome newbie like me would overlook? Is there an issue with newer ESPhome versions? Or should I assume the C1101 chip is dead? Is there any way to check that there is successful communication to the C1101? I might be able to borrow a VNA scope from work to check if there are radio signals sent but I'm not sure if that could tell me anything

**Comment by CoMPaTech** [comment]
I've not 're'installed it, but I keep mine up to date even to the latest version released yesterday. Basically, just like one of the wall controllers, once you send the 'pair' command you should hear the engine rev up. The basic procedure was something like 'disconnect engine from mains' - 'reconnect' - 'within 30 seconds send pairing command (from your esp)'.

**Comment by AllardKatan** [comment]
> I've not 're'installed it, but I keep mine up to date even to the latest version released yesterday. Basically, just like one of the wall controllers, once you send the 'pair' command you should hear the engine rev up. The basic procedure was something like 'disconnect engine from mains' - 'reconnect' - 'within 30 seconds send pairing command (from your esp)'.

Yes, I've done that, but it never actually pairs, revs up the engine or reacts in any other way to commands I send from the ESP...

**Comment by CoMPaTech** [comment]
The older model was 433 Mhz, the newer installed was an 866 Mhz one that works with the C1101. Besides indeed scoping it or in proximity to a transceiver to see/hear that it actually would transmit I'm not aware of any proper way to debug it. Mine's still connected to a (now) pretty ancient ESP, but even that shouldn't really matter.
It might be the type of unit and/or something changed if you have a newer one w.r.t. adding 'switches' (which is what the ESP would be for the engine)? Might be worth pinging someone in one of the closed issues if they have any idea. Maybe I can think of something overnight, but nothing pops up as a clear indication just yet.

**Comment by AllardKatan** [comment]
866 MHz is what I should have indeed, and the description you gave is the same as what is in my manual for adding new remotes. Thanks for your quick replies! Hopefully someone else has the golden idea
